lib/Metrc/client.rb

```diff
@@ -163,7 +164,11 @@ def destroy_plant_batches(license_number, resources)
     end
 
     def create_plant_batch_package(license_number, resources)
-      api_post("/plantbatches/v1/create/plantings?licenseNumber=#{license_number}", body: resources.to_json)
+      uri = '/plantbatches/v1/createpackages'
```

Comment:
I think it's too soon to know if we'll need different urls based on an array of states, or if there will be some kind of condition. This might be premature optimization.

A simpler

```
url = if configuration.state == 'ca'
  '/plantbatches/v1/create/plantings`
else
  '/plantbatches/v1/createpackages`
end
```

is more legible, and better conveys the intention of the logic and our understanding as of today.